positional/benches/enum.rs

```diff
@@ -85,7 +85,7 @@ fn deserialize_enum(c: &mut Criterion) {
 criterion_group! {
     name = benches;
     // This can be any expression that returns a `Criterion` object.
-    config = Criterion::default().sample_size(60);
+    config = Criterion::default().sample_size(30);
```

Comment:
I didn't expect to see changes for the benchmarks in this PR, but since it's such a small change, I don't mind.
Why are we reducing the sample size, though?

Reply:
because criterion needs a sample size small enough to be completed in 3 seconds. My PC is pretty old and bloated, so maybe on better PCs this is not needed. Anyway it's just useful for predicting the time needed to complete the test